index.md

```diff
@@ -6,7 +6,11 @@ redirect_from:
   - "/2019/"
 ---
 
-Our Networks is a conference about the past, present, and future of building our own network infrastructures. Now in its third year, we are hard at work planning for 2019.
+Our Networks 2019 explores the collective care and maintenance of alternative networking practices&mdash;new protocols, peer-to-peer connections, offline-first computing, or community-based governance and ownership. Building on last year, where we asked how to move beyond solitary practices of do it yourself (DIY), in 2019 we explore what critical and creative practices offer **RE: Infrastructures** of alternatives.
+
+How can these emerging **Infrastructures of <span id="word"></span>** provide spaces to imagine radically different futures, resist easy co-option by the forces of late-stage capitalism, and work in solidarity as plural networks seeking emancipatory change? To answer this we draw on insights from organizers, technologists, artists, and scholars to ensure the focus is not only on the technical aspects of infrastructure, but also the social and cultural. Their diverse perspectives suggest ways we can make space for livelihoods, solidarity, and survival; draw on history and place to bring our fullest imaginations to this moment; and open ways to think these questions together as we build transitional forms.
```

Comment:
Can we restrict the flow that results from the typing effect to a single line like this? I tried to read the paragraph and got dizzy 😭 

```suggestion
How can these emerging **Infrastructures of <span id="word"></span>** provide spaces to imagine  
radically different futures, resist easy co-option by the forces of late-stage capitalism, and work in solidarity as plural networks seeking emancipatory change? To answer this we draw on insights from organizers, technologists, artists, and scholars to ensure the focus is not only on the technical aspects of infrastructure, but also the social and cultural. Their diverse perspectives suggest ways we can make space for livelihoods, solidarity, and survival; draw on history and place to bring our fullest imaginations to this moment; and open ways to think these questions together as we build transitional forms.
```

Reply:
This change causes some v. strange wrap issues on mobile and non. I think we should revisit the decision to add in a line break. Maybe adjust the animation speed? @garrying ?